docs/docs/writing-documentation-with-docz.md

```diff
@@ -0,0 +1,163 @@
+global.__PATH_PREFIX__ = ""
+
+export default ({ children }) => children
+```
+
+> You are essentially creating a dummy wrapper that does nothing else than making sure that `global.__PATH_PREFIX__` is defined on every page.
+
+Create a new file called `doczrc.js` in the root directory of your Gatsby project, and add the following content:
+
+```js:title=doczrc.js
+import merge from "webpack-merge"
+import { css } from "docz-plugin-css"
+
+export default {
+  title: "Docz with Gatsby",
+  // Add CSS support in case you use them in your Gatsby project
+  plugins: [css()],
+  // highlight-next-line
+  // Wrapper used to inject some global variable mocks
+  wrapper: "docz/wrapper.js",
+  modifyBundlerConfig: config => {
+    const gatsbyNecessaryConfig = {
+      module: {
+        rules: [
+          {
+            // Transpile Gatsby module because Gatsby includes un-transpiled ES6 code.
+            // Ignore .json files because they fail to be parsed
+            exclude: [/node_modules\/(?!(gatsby)\/)/, /\.json$/],
+            use: [
+              {
+                // use installed babel-loader which is v8.0-beta (which is meant to work with @babel/core@7)
+                loader: "babel-loader",
+                options: {
+                  // use @babel/preset-react for JSX and env (instead of staged presets)
+                  presets: ["@babel/preset-react", "@babel/preset-env"],
+                  plugins: [
+                    // use @babel/plugin-proposal-class-properties for class arrow functions
+                    "@babel/plugin-proposal-class-properties",
+                    // use @babel/plugin-syntax-dynamic-import for dynamic import support
+                    "@babel/plugin-syntax-dynamic-import",
+                  ],
+                },
+              },
+            ],
+          },
+        ],
+      },
+    }
+
+    return merge(gatsbyNecessaryConfig, config)
+  },
+}
+```
+
+Once you have this configured you should run Docz to ensure it can start up properly. You should see by default a _Page Not Found_ page, this is fine as you haven't created any `mdx` files yet. To run Docz:
+
+```shell
+npm run docz:dev
+```
+
+If Docz builds successfully you should be able to navigate to `http://localhost:3000` and see the default _Page Not Found_ page.
+
+## Writing documentation
+
+Docz searches your directory for `mdx` files and renders them. Let's add your first documentation page by creating a file `index.mdx` in the root directory of your Gatsby project.
+
+```mdx:title=index.mdx
+---
+name: Getting started
+route: /
+---
+
+# Getting Started
+
+This is the start of an amazing Docz site
+```
+
+This is a very simple documentation page without much going on, but let's spice things up by adding and rendering a Gatsby component. Assuming you have a header component in your components folder which does not rely on `StaticQuery` or `graphql`, you can add:
```

Comment:
Could you add a note on what to do if your component does use `StaticQuery` or `graphql`? Even if the note is just 'refactor your component to avoid using those things' I think that's still useful,